[Bug]: Can not compile Termsequel on Debian 10

### Contact Details

_No response_

### What happened?

Can not execute with success **`cmake --build build`**.    

There is a conflict with glibc implementation ...  Since we use: `sys/stat` and `/linux/stat.h` we are declaring twice some **stat** structures ...

### Version

1.1

### What operating system are you seeing the problem on?

Linux

### Relevant log output

```shell
$ cmake --build ./build/
Scanning dependencies of target termsequel
[ 12%] Building CXX object CMakeFiles/termsequel.dir/src/system.cpp.o
In file included from /home/sgtcortez/Documents/Projects/Termsequel/src/system.cpp:29:
/usr/include/linux/stat.h:56:8: error: redefinition of ‘struct statx_timestamp’
 struct statx_timestamp {
        ^~~~~~~~~~~~~~~
In file included from /usr/include/x86_64-linux-gnu/sys/stat.h:446,
                 from /home/sgtcortez/Documents/Projects/Termsequel/src/system.cpp:15:
/usr/include/x86_64-linux-gnu/bits/statx.h:25:8: note: previous definition of ‘struct statx_timestamp’
 struct statx_timestamp
        ^~~~~~~~~~~~~~~
In file included from /home/sgtcortez/Documents/Projects/Termsequel/src/system.cpp:29:
/usr/include/linux/stat.h:99:8: error: redefinition of ‘struct statx’
 struct statx {
        ^~~~~
In file included from /usr/include/x86_64-linux-gnu/sys/stat.h:446,
                 from /home/sgtcortez/Documents/Projects/Termsequel/src/system.cpp:15:
/usr/include/x86_64-linux-gnu/bits/statx.h:36:8: note: previous definition of ‘struct statx’
 struct statx
        ^~~~~
make[2]: *** [CMakeFiles/termsequel.dir/build.make:63: CMakeFiles/termsequel.dir/src/system.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:73: CMakeFiles/termsequel.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```


### Relevant print output

![image](https://user-images.githubusercontent.com/33197461/132992355-8eb5e1c6-eda3-4946-be6c-5cb21c87568e.png)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 
 cmake_minimum_required(VERSION 3.10)
 
-project(termsequel VERSION 1.1.1)
+project(termsequel VERSION 1.1.2)
 
 configure_file(termsequel.hpp.in termsequel.hpp)
 

--- a/Changelog.md
+++ b/Changelog.md
@@ -86,3 +86,11 @@ Contains information about the versions of Termsequel
         | Cant use on systems with glibc older than 2.28 | [ISSUE](https://github.com/sgtcortez/Termsequel/issues/82) | [PR](https://github.com/sgtcortez/Termsequel/pull/83) | Now, we call **stax** system call directly |         
         | Creation Date overwrite Last Modification | [ISSUE](https://github.com/sgtcortez/Termsequel/issues/84) | [PR](https://github.com/sgtcortez/Termsequel/pull/85) | Since, the add of statx syscall, we overwrite the last modification date|
         | Star do not include CREATION_DATE  | [ISSUE](https://github.com/sgtcortez/Termsequel/issues/86) | [PR](https://github.com/sgtcortez/Termsequel/issues/87) | When select with star, it do not include the new creation date column | 
+
+- Release 1.1.2
+
+    - Bug Fixes
+
+        | Bug | Issue | Pull Request |
+        | :-:   | :-: | :-: | :-: |    
+        | Double declaration of statx structure | [ISSUE](https://github.com/sgtcortez/Termsequel/issues/93) | [PR](https://github.com/sgtcortez/Termsequel/pull/94) |

--- a/src/system.cpp
+++ b/src/system.cpp
@@ -12,7 +12,6 @@
 #include <sstream>
 #include <streambuf>
 #include <string>
-#include <sys/stat.h>
 #include <sys/types.h>
 #include <vector>
 #include <time.h>
@@ -84,6 +83,7 @@ constexpr static const char FILE_SEPARATOR = '/';
 
 #include <io.h> // _findfirst _findnext
 #include <windows.h>  // GetFullPathName
+#include <sys/stat.h>
 
 #define IS_DIRECTORY(bitset) (bitset & _S_IFDIR)
 #define FILE_TYPE_MASK _S_IFMT

